scripts/prepare-aap.sh

```
@@ -33,17 +35,16 @@ fi
 # Store the token in a Kubernetes secret
 oc create secret generic osac-aap-api-token \
     --from-literal=token="${AAP_TOKEN}" \
-    -n "${INSTALLER_NAMESPACE}" \
+    -n ${INSTALLER_NAMESPACE} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)